Unit Tests/SMLHelper/LanguagePatcher.cs

```diff
@@ -6,7 +6,7 @@
 namespace QModManager.Tests.SMLHelper
 {
     [TestFixture]
-    public class LanguagePatcherTests
+    public class LanguagePatcher
```

Comment:
Any reason why this needed to be renamed?  This is a collection of unit tests.  
Do you have objections against the naming convention "<class being tested>Tests" for these TestFixtures?

Reply:
Nope, just wanted to unify this with QMM's tests which do not include Tests at the end. I guess I'll do the other way around.